apps/web/pages/organizations/[id]/subscriptionStatusOrganizationLink/subscriptionStatusOrganizationLink.tsx

```diff
@@ -0,0 +1,36 @@
+import { useRouter } from 'next/router'
+import { FaCircleCheck, FaCircleXmark, FaTriangleExclamation } from 'react-icons/fa6'
+
+import { StatusLink } from './statusLink'
+
+export const SubscriptionStatusOrganizationLink = () => {
+  const router = useRouter()
+  const { status } = router.query
+
+  switch (status) {
+    case 'subscriptionSuccess':
+      return (
+        <StatusLink className="alert-success">
+          <FaCircleCheck className="text-xl" />
```

Comment:
`SubscriptionStatusOrganizationLink` should not decide which icon and which CSS rules the alert should have.

Take a look at #951. This separates the implementation details of the alert from the usage.